Build Failure on GHC 8.4.1

ODBC fails to build with GHC 8.4.1 with the following error:
```
src/Database/ODBC/SQLServer.hs:211:13: error:
    • No instance for (Semigroup Query)
        arising from the 'deriving' clause of a data type declaration
      Possible fix:
        use a standalone 'deriving instance' declaration,
          so you can specify the instance context yourself
    • When deriving the instance for (Monoid Query)
    |
211 |   deriving (Monoid, Eq, Show, Typeable, Ord, Generic, Data)
    |  
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -17,4 +17,4 @@ script:
   - echo "Running docker for commit $TRAVIS_COMMIT"
   - echo $TRAVIS_COMMIT > travis_commit.txt
   - docker build . --tag stack-test
-  - docker run --net=host -e "ODBC_TEST_CONNECTION_STRING=\"DRIVER=ODBC Driver 17 for SQL Server;SERVER=127.0.0.1;Uid=SA;Pwd=Passw0rd\"" -w /odbc stack-test:latest stack test
+  - docker run --net=host -e "ODBC_TEST_CONNECTION_STRING=\"DRIVER=ODBC Driver 17 for SQL Server;SERVER=127.0.0.1;Uid=SA;Pwd=Passw0rd\"" -w /odbc stack-test:latest sh test/test-ghcs

